packages/e2e/cypress.config.ts

```diff
@@ -20,7 +20,6 @@ export default defineConfig({
             'analytics.lightdash.com',
         ],
         trashAssetsBeforeRuns: true,
-        videoUploadOnPasses: false,
```

Comment:
double check

Reply:
don't need it anymore with cypress upgrade

Reply:
![CleanShot 2024-01-12 at 17 33 49@2x](https://github.com/lightdash/lightdash/assets/962095/0733daf8-2025-49d7-a403-3a6bb9bab938)
its fine I can see a failed test video here

Reply:
nope! it's not fine. no videos only screenshots 😫 
![CleanShot 2024-01-12 at 17 34 52@2x](https://github.com/lightdash/lightdash/assets/962095/11c631e7-50e5-4a2d-b0fa-7f54f99b42a4)

Reply:
That's intended though! We removed the option to upload videos a few months(?) ago

Reply:
fixed: 
![CleanShot 2024-01-12 at 18 35 37@2x](https://github.com/lightdash/lightdash/assets/962095/87c2be09-f21b-4ddd-a0fb-d9ecd53a7862)

Reply:
@joaoviana I remember it being enabled. no?

Reply:
https://lightdash.slack.com/archives/C02GQKJK84Q/p1705071638465379

should be enabled ✅